### DISCUSSION TITLE
Welcome to kepler.gl Discussions!

### DISCUSSION THREAD

**Discussion by ibgreen** [keplergl/kepler.gl#2499] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by malema24** [comment]
Hello everyone precisely I am a  fullstuckfront-end webdesigner.....hence I do design apk also hostingwebsite..........I am a mechatronics engineer still a student studying hence I self study myself alot I work 20 hours each and everyday only for advancing my coding skills

**Comment by danielquintero2** [comment]
Could we get kepler gl 3.0.0 on Jupyter Notebooks soon? pip version is outdated. Thanks!! this tool is awesome
